tests/flytekit/unit/core/test_type_hints.py

```diff
@@ -1006,3 +1007,15 @@ def squared(value: int) -> typing.List[typing.Dict[str, int]]:
     def compute_square_wf(input_integer: int) -> typing.List[typing.Dict[str, int]]:
         compute_square_result = squared(value=input_integer)
         return compute_square_result
+
+
+def test_secrets():
+    @task(secret_requests=["my_secret"])
```

Comment:
Should this be `Secret("my_secret")`

Reply:
aah yes it should be, and thus in this case maybe i should do a check on this

Reply:
done